BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -25,52 +24,43 @@ struct TransactionConfirmationView: View {
   }
   
   @State private var viewMode: ViewMode = .transaction
-  @State private var activeTransactionId: BraveWallet.TransactionInfo.ID = "" {
-    didSet {
-      confirmationStore.fetchDetails(for: activeTransaction)
-    }
-  }
   
   private func next() {
-    if let index = transactions.firstIndex(where: { $0.id == activeTransactionId }) {
-      var nextIndex = transactions.index(after: index)
-      if nextIndex == transactions.endIndex {
+    if let index = confirmationStore.transactions.firstIndex(where: { $0.id == confirmationStore.activeTransactionId }) {
+      var nextIndex = confirmationStore.transactions.index(after: index)
+      if nextIndex == confirmationStore.transactions.endIndex {
         nextIndex = 0
       }
-      activeTransactionId = transactions[nextIndex].id
+      confirmationStore.activeTransactionId = confirmationStore.transactions[nextIndex].id
     } else {
-      activeTransactionId = transactions.first!.id
+      confirmationStore.activeTransactionId = confirmationStore.transactions.first!.id
     }
   }
   
   private func rejectAll() {
-    for transaction in transactions {
+    for transaction in confirmationStore.transactions {
       confirmationStore.reject(transaction: transaction)
     }
   }
   
-  private var activeTransaction: BraveWallet.TransactionInfo {
-    transactions.first(where: { $0.id == activeTransactionId }) ?? transactions.first!
-  }
-  
```

Comment:
Why not keep this but instead return `confirmationStore.activeTransaction` so you can keep the same code instead of doing `activeTransaction` → `confirmationStore.activeTransaction` everywhere?